### PR TITLE
Add passed test case for ngo

### PR DESCRIPTION
**Pull request by daan3601**

### CODE REVIEW

File: occlum/run_occlum_passed_tests.sh [hunk 1]
```diff
@@ -24,16 +38,35 @@ get_blocklist_subtests(){
 run_one_test(){
 	ret=""
 	if [ -f $TEST_BIN_DIR/$1 ]; then
-		get_blocklist_subtests $1
-		occlum exec /bin/$1 --gtest_filter=-$FILTER
+		printf "$GREEN running $1$NC"
```
Comment (jessehui) [inline occlum/run_occlum_passed_tests.sh]: Please change line here too.

File: occlum/run_occlum_passed_tests.sh [hunk 2]
```diff
@@ -3,13 +3,27 @@
...
 TEST_LIST=../occlum_test_list.txt
 TESTS=0
 PASSED_TESTS=0
 RED='\033[0;31m'
+GREEN='\033[0;32m'
 NC='\033[0m'
 RESULT=0
 FILTER=""
+NGO_FILTER=""
+Way=$*
```
Comment (jessehui) [inline occlum/run_occlum_passed_tests.sh]: Please follow the naming convention.

File: occlum/run_occlum_passed_tests.sh [hunk 3]
```diff
@@ -24,16 +38,35 @@ get_blocklist_subtests(){
...
 	else
 		echo "Warning: test does not exit"
 		ret=1
 	fi
 	return $ret
 }
 
+if [ -z $Way ];then
+	printf "$GREEN Running test cases passed in occlum $NC\n"
+else
```
Comment (jessehui) [inline occlum/run_occlum_passed_tests.sh]: Maybe `elif` here too?

Comment (daan3601) [reply occlum/run_occlum_passed_tests.sh:62]: Here it is judged whether there are parameters passed in, if not run occlum by default

File: occlum/run_occlum_passed_tests.sh (hunk 1 above)
Comment (daan3601) [reply occlum/run_occlum_passed_tests.sh]: Modification complete

File: occlum/run_occlum_passed_tests.sh (hunk 2 above)
Comment (daan3601) [reply occlum/run_occlum_passed_tests.sh]: Modification complete

File: occlum/run_occlum_passed_tests.sh (hunk 3 above)
Comment (daan3601) [reply occlum/run_occlum_passed_tests.sh]: Here we mainly judge whether the incoming parameters are wrong

File: occlum/run_occlum_passed_tests.sh [hunk 4]
```diff
@@ -24,16 +38,33 @@ get_blocklist_subtests(){
...
 		ret=1
 	fi
 	return $ret
 }
 
+if [ -z $OPERATION_MODE ];then
+	printf "$GREEN Running test cases passed in occlum $NC\n"
+elif [ $OPERATION_MODE != "ngo" ];then
+	echo The parameter is wrong, you can use '"./run_occlum_passed_tests.sh ngo"' to run the tests passed in ngo
+		exit -1
```
Comment (jessehui) [inline occlum/run_occlum_passed_tests.sh]: This indent is wrong

File: occlum/run_occlum_passed_tests.sh [hunk 5]
```diff
@@ -24,16 +38,33 @@ get_blocklist_subtests(){
...
 		echo "Warning: test does not exit"
 		ret=1
 	fi
 	return $ret
 }
 
+if [ -z $OPERATION_MODE ];then
+	printf "$GREEN Running test cases passed in occlum $NC\n"
+elif [ $OPERATION_MODE != "ngo" ];then
+	echo The parameter is wrong, you can use '"./run_occlum_passed_tests.sh ngo"' to run the tests passed in ngo
```
Comment (jessehui) [inline occlum/run_occlum_passed_tests.sh]: It is better to quote the msg for `echo`

Comment (daan3601) [reply occlum/run_occlum_passed_tests.sh]: Modification has been completed

File: occlum/run_occlum_passed_tests.sh (hunk 4 above)
Comment (daan3601) [reply occlum/run_occlum_passed_tests.sh]: Modification has been completed

File: occlum/run_occlum_passed_tests.sh (hunk 5 above)
Comment (jessehui) [reply occlum/run_occlum_passed_tests.sh]: I mean line 63.

Comment (daan3601) [reply occlum/run_occlum_passed_tests.sh]: Modification has been completed

File: occlum/occlum_workspace/.__occlum_status [hunk 6]
```diff
@@ -0,0 +1 @@
+initialized
```
Comment (jessehui) [inline occlum/occlum_workspace/.__occlum_status]: occlum_workspace shouldn't be included.

Comment (daan3601) [reply occlum/occlum_workspace/.__occlum_status]: Generated during testing, has been deleted

**Review by jessehui — APPROVED** [review]
LGTM. Thank you for your contribution.

**Review by guzongmin — APPROVED** [review]
LGTM

**Pull request merged**